build.sbt

```diff
@@ -0,0 +1,36 @@
+import Dependencies._
```

Comment:
Wondering if it makes sense to put all the Notify code in a subfolder, so it's not mixed in with all the regular Keycloak code?

Reply:
Are talking about including the build.sbt etc in a separate sub folder:
```
tdr-auth-server
+ govuk-notify-spi
++ project
+++ Dependencies
+++ plugins
++ build.sbt
++ src
+++ etc 
```
Or would the `build.sbt` still be in the main folder

Reply:
The former. I was just looking at your branch, and it's not clear which files/folders are part of the Notify build, and which are separate (e.g. the themes, environment properties).

Reply:
Restructured